coconut/configuration/configuration.go

```diff
@@ -343,4 +427,42 @@ func GetListOfComponentsAndOrWithTimestamps(keys []string, keyPrefix string, use
 		}
 	}
 	return components, nil, nonZero
-}
\ No newline at end of file
+}
+
+func drawTableHistoryConfigs(headers []string, history []string, max int, o io.Writer) {
+	table := tablewriter.NewWriter(o)
+	if len(headers) > 0 {
+		table.SetHeader(headers)
+	}
+	table.SetBorder(false)
+	table.SetColMinWidth(0, max)
+
+	for _, value := range history {
+		component, entry, timestamp := GetComponentEntryTimestamp(value)
+		prettyTimestamp, err := GetTimestampInFormat(timestamp, time.RFC822)
+		if err != nil {
+			prettyTimestamp = timestamp
+		}
+		configName := red(component) + "/" + blue(entry) + "@" + timestamp
+		table.Append([]string{configName, prettyTimestamp})
+	}
+	table.Render()
+}
+
+func GetComponentEntryTimestamp(key string)(string, string, string) {
+	key = strings.TrimPrefix(key, componentsPath)
+	key = strings.TrimPrefix(key, "/'")
+	key = strings.TrimSuffix(key, "/")
+	elements := strings.Split(key, "/")
+	return elements[0], elements[1], elements[2]
+}
+
+func GetMaxLenOfKey(keys []string) (maxLen int){
```

Comment:
Are these kind of functions ever used outside this configuration package? If not, they should be private.

Reply:
The plan is to move them to configuration_util. Similar to what control_util is.
Also I need them to be exported for testing purposes. Thus, I made them exportable from start